packages/core/src/domain/configuration/transportConfiguration.spec.ts

```diff
@@ -141,11 +141,46 @@ describe('transportConfiguration', () => {
     })
   })
 
+  describe('datacenter', () => {
+    ;[
+      { site: 'datadoghq.eu', datacenter: 'eu1.prod.dog' },
+      { site: 'datadoghq.com', datacenter: 'us1.prod.dog' },
+      { site: 'us3.datadoghq.com', datacenter: 'us3.prod.dog' },
+      { site: 'us5.datadoghq.com', datacenter: 'us5.prod.dog' },
+    ].forEach(({ site, datacenter }) => {
+      it(`should be set as tag for site ${site} in the logs and rum endpoints`, () => {
+        const configuration = computeTransportConfiguration({ clientToken, site }, buildEnv)
+        expect(decodeURIComponent(configuration.rumEndpointBuilder.build())).toMatch(
+          `&ddtags=(.*),datacenter:${datacenter}`
+        )
+        expect(decodeURIComponent(configuration.logsEndpointBuilder.build())).toMatch(
+          `&ddtags=(.*),datacenter:${datacenter}`
+        )
+      })
+
+      it(`should be set as tag for site ${site} in the logs and rum endpoints replicas`, () => {
+        const configuration = computeTransportConfiguration({ clientToken, site, replica: { clientToken } }, buildEnv)
+
+        expect(decodeURIComponent(configuration.replica!.rumEndpointBuilder.build())).toMatch(
+          `&ddtags=(.*),datacenter:${datacenter}`
+        )
+        expect(decodeURIComponent(configuration.replica!.logsEndpointBuilder.build())).toMatch(
+          `&ddtags=(.*),datacenter:${datacenter}`
+        )
+      })
+    })
+
+    it('should not be set as tag for other datacenters in the logs and rum endpoints', () => {
```

Comment:
should not we set it for us1.prod.dog in order to have all rum events in us1 tagged with a datacenter?
it seems to be the case for other data sources